packages/back-office-subscribers/lib/prisma.js

```diff
@@ -0,0 +1,38 @@
+const { PrismaClient } = require('@prisma/client');
+const isProduction = process.env.NODE_ENV === 'production';
+
+let prismaClientInstance;
+
+const createPrismaClient = () => {
+	const logOption = isProduction
+		? ['query', 'info', 'warn', 'error']
+		: [{ emit: 'event', level: 'query' }, 'info', 'warn', 'error'];
+
+	if (!prismaClientInstance) {
+		prismaClientInstance = new PrismaClient({
+			log: logOption
+		});
+	}
+
+	if (!isProduction) registerQueryLogger(prismaClientInstance);
```

Comment:
I think we can get away with the default logging behaviour here. So I would be inclined to remove `registerQueryLogger` and keep the `logOption`s the same between all environments. It only differs to enable this custom logging function (`emit: 'event'`) anyway.